apiserver/plane/db/models/__init__.py

```
@@ -87,7 +84,7 @@
 
 from .importer import Importer
 
-from .page import Page, PageLog, PageFavorite, PageLabel
+from .page import Page, PageLog, PageLabel
```

Ruff

Redefinition of unused `Page` from line 53

Remove definition: `Page`

(F811)